pkg/apis/messaging/v1/subscription_types.go

```diff
@@ -181,3 +181,12 @@ func (s *Subscription) GetUntypedSpec() interface{} {
 func (s *Subscription) GetStatus() *duckv1.Status {
 	return &s.Status.Status
 }
+
+func (s *Subscription) GetCrossNamespaceRef() duckv1.KReference {
+	return duckv1.KReference{
+		APIVersion: s.APIVersion,
+		Kind:       s.Kind,
+		Name:       s.Name,
+		Namespace:  s.Namespace,
```

Comment:
Similarly, shouldn't this be the `channel` reference, rather than pointing at the subscription again?